### DISCUSSION TITLE
#44

### DISCUSSION THREAD

**Discussion by himanshubhatt1** [cssbattle/forum#26] [General]
Please help me for #44 css battle , i stucked in 99.9%
<style>*{margin:60 120;background:repeating-linear-gradient(#f3ac3c 0 20px,#0000 1q 40px)no-repeat#1a4341}*>*{margin:-60 -270 -60 130;border-radius:200px;background:#1a4341;-webkit-box-reflect:left 106q

**Comment by duarch** [comment]
I have a similar issue we're are discussing in this topic 👉 #12
